apps/src/lib/levelbuilder/course-editor/CourseEditor.js

```diff
@@ -273,20 +273,25 @@ class CourseEditor extends Component {
           </label>
         </CollapsibleEditorSection>
 
-        <CollapsibleEditorSection title="Teacher Resources">
+        <CollapsibleEditorSection title="Resources Dropdowns">
           {this.props.migratedTeacherResources && (
             <input
               type="hidden"
               name="resourceIds[]"
               value={this.props.migratedTeacherResources.map(r => r.id)}
             />
           )}
+          {this.props.studentResources && (
+            <input
+              type="hidden"
+              name="studentResourceIds[]"
```

Comment:
this line tipped me off that the course edit page is using a haml form. I'm not sure if people spend enough time on the course edit page for it to be a big priority to change this, but if you run into issues which make you think that we should prioritize this please let me know. :-)

Reply:
The only thing I noticed was the fact that I couldn't "save and continue" but otherwise it didn't seem like a big issue.